src/terminologies/mdm-code-set.resource.ts

```diff
@@ -250,4 +251,20 @@ export class MdmCodeSetResource extends MdmModelDomainResource {
     const url = `${this.apiEndpoint}/admin/codeSets/${codeSetId}/undoSoftDelete`;
     return this.simplePut(url, {}, options);
   }
+
+  /**
+   * `HTTP PUT` - Copy a code set.
+   *
+   * @param codeSetId The unique identifier of the code set to copy.
+   * @param CopyContainerPayload The payload to pass to the request when copying the code set.
+   * @returns The result of the `PUT` request.
+   *
+   * `200 OK` - will return a {@link CodeSetDetailResponse} containing a {@link CodeSetDetail} object.
+   */
+  copy(codeSetId: Uuid,
+    copyContainerPayload: CopyContainerPayload
```

Comment:
More consistency things:

- [ ] Move `target...` and `source...` properties into function parameters
- [ ] `copyContainerPayload` is not the correct name for this, this implies we're copying a container (like a folder) whereas this is for a model. `CopyModelPayload` would make more sense

Reply:
targetFolder is a payload for these three